src/main/java/fi/metatavu/metaform/server/persistence/dao/AuditLogEntryDAO.java

```diff
@@ -0,0 +1,109 @@
+package fi.metatavu.metaform.server.persistence.dao;
+
+import fi.metatavu.metaform.client.model.AuditLogEntryType;
+import fi.metatavu.metaform.server.persistence.model.*;
+import org.slf4j.Logger;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.persistence.TypedQuery;
+import javax.persistence.criteria.*;
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * DAO class for AuditLogEntry entity
+ *
+ * @author Katja Danilova
+ */
+@ApplicationScoped
+public class AuditLogEntryDAO extends AbstractDAO<AuditLogEntry> {
+
+  /**
+   * Creates AuditLogEntry
+   *
+   * @param id uuid of log entry
+   * @param metaform metaform
+   * @param userId userId
+   * @param time time
+   * @param replyId replyId
+   * @param attachmentId attachmentId
+   * @param message message
+   * @return created auditlogentry
+   */
+  public AuditLogEntry create(UUID id, Metaform metaform, UUID userId, OffsetDateTime time, AuditLogEntryType auditLogEntryType, UUID replyId,
+                              UUID attachmentId, String message){
+    AuditLogEntry auditLogEntry = new AuditLogEntry();
+    auditLogEntry.setId(id);
+    auditLogEntry.setMetaform(metaform);
+    auditLogEntry.setUserId(userId);
+    auditLogEntry.setTime(time);
+    auditLogEntry.setLogEntryType(auditLogEntryType);
+    auditLogEntry.setReplyId(replyId);
+    auditLogEntry.setAttachmentId(attachmentId);
+    auditLogEntry.setMessage(message);
+
+    return persist(auditLogEntry);
+  }
+
+    /**
+     * Gets audit log entries by replies, user id, created before and after parameters
+     *
+     * @param metaform replyId
+     * @param replyId replyId
+     * @param userId userId
+     * @param createdBefore created before
+     * @param createdAfter created after
+     * @return list of AuditLogEntry
+     */
+    public List<AuditLogEntry> listAuditLogEntries(Metaform metaform, UUID replyId, UUID userId, OffsetDateTime createdBefore, OffsetDateTime createdAfter) {
```

Comment:
In DAO method it's just list
```suggestion
    public List<AuditLogEntry> list(Metaform metaform, UUID replyId, UUID userId, OffsetDateTime createdBefore, OffsetDateTime createdAfter) {
```